server/py/services/api/tests/unit/utils/projects/test_follower_member.py

```diff
@@ -466,7 +466,9 @@ async def test_list_project_summaries_fails_to_list_pipeline_runs(
         project,
     )
     framework.utils.singletons.db.get_db().get_project_resources_counters = (
-        unittest.mock.AsyncMock(return_value=tuple({project_name: i} for i in range(9)))
+        unittest.mock.AsyncMock(
+            return_value=tuple({project_name: i} for i in range(12))
+        )
```

Comment:
why?

Reply:
because now get_project_resources_counters returns 12 counters and not 9